src/Components/test/testassets/GlobalizationWasmApp/GlobalizationWasmApp.csproj

```diff
@@ -9,6 +9,13 @@
     <GenerateResxSource>false</GenerateResxSource>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TestTrimmedApps)' == 'true'">
+    <StaticWebAssetBasePath>/subdir</StaticWebAssetBasePath>
```

Comment:
Why does this only apply to trimmed apps?

Reply:
This weirds out our in-place tests if we configure the StaticWebAssetBasePath. It's a result of how we run the test - by specifying a base path to the host. I tried changing it but it seemed like a messy ball to undo.

Btw, I'm holding off on merging this PR because it conflicts with my hot reload work. We're expressly removing a lot of the hot reload APIs during trimming which poses a problem with the approach here.